Update chopin payment tutorial (#61)
apiobuild/apiobuild.github.io: content/blog/create-an-online-store-for-free.md

#### Customization 

> The website is fully customizable. As of the [website appearance](https://apiobuild.com/blog/how-to-configure-chopin-store/#store-information), you can create an online store that aligns with your brand by providing **logo, store description, background color, and background image** on [Chopin\'s configure tab](https://telescope.apiobuild.com/app/chopin). More customization options are also available through simple HTML blocks.

Reviewer: ```suggestion
The website is fully customizable. As of the [website appearance](https://apiobuild.com/blog/how-to-configure-chopin-store/#store-information), you can create an online store that aligns with your brand by providing **logo, store description, background color, and background image** in [Chopin app](https://telescope.apiobuild.com/app/chopin). More customization options are also available through simple HTML blocks.
```

<img src="/images/blog/customization.png" class="post-img">

#### Communication 

> There are a few communication channels designed for you and your customers to create flawless conversation. Chopin enables social plug-ins so your customers can easily reach you via **[Phone, Email, Instagram, Facebook, and Line](https://apiobuild.com/blog/how-to-configure-chopin-store/#contact-information)**. You also have the option to add **[announcement](https://apiobuild.com/blog/how-to-configure-chopin-store/#store-information)** to broadcast important updates or discount information.

Reviewer: ```suggestion
There are a few communication channels designed for you and your customers to create a flawless conversation experience. Chopin enables social plug-ins so your customers can easily reach you via **[Phone, Email, Instagram, Facebook, and Line](https://apiobuild.com/blog/how-to-configure-chopin-store/#contact-information)**. You also have the option to add **[announcement](https://apiobuild.com/blog/how-to-configure-chopin-store/#store-information)** to broadcast important updates or discount information.
```
